InformationScripting/src/queries/ScopedArgumentQuery.cpp

```diff
@@ -53,4 +53,15 @@ QString ScopedArgumentQuery::argument(const QString& argName) const
 	return argParser_->value(argName);
 }
 
+bool ScopedArgumentQuery::isArgumentSet(const QString& argName) const
+{
+	return argParser_->isSet(argName);
+}
+
+Model::SymbolMatcher ScopedArgumentQuery::matcherFor(const QString& text)
```

Comment:
Actually this should belong to the `SymbolMatcher` itself. It could be useful to add in general a few static functions to construct common and useful matchers. We can call this one `static Matcher guessMatcher(const QString& s)`.